specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -200,11 +200,12 @@ export function IntegratedRecordSelector({
       relationship={relationship}
       onAdd={handleAdd}
       onDelete={handleDelete}
-      onSlide={(index): void =>
+      onSlide={(index): void => {
+        focusFirstField();
```

Comment:
are you sure you want to call focusFirstField here?
I belive we talked about it and decided that it's a bad idea.

I.e if you want to go to the record that is 5 records to the right of current one, every time your press next it will move your focus and you will have to move it back - not very keyboard friendly.

Perharps the best solution is to call focusFirstField() if the next/previous button was pressed with a mouse, and don't call focusFirstField() if next/previous was pressed with a keyboard enter